src/entities/document/components/DocumentNewForm/storage.ts

```
@@ -0,0 +1,37 @@
+        return parsed ?? {};
+    } catch {
+        return {};
+    }
+}
+
+export function saveCoverLetterDraft(draft: CoverLetterDraft): void {
+    if (typeof window === "undefined" || !("localStorage" in window)) return;
+    try {
+        const sanitized =
+            draft?.questions
+                ? { ...draft, questions: draft.questions.map(({ isOpen, ...q }) => q) }
```

ESLint

[error] 'isOpen' is defined but never used.

(@typescript-eslint/no-unused-vars)